platform/testing/liquid-templates/README.md

```diff
@@ -5,13 +5,158 @@ The Liquid Template Unit Testing Framework was created to replace Cypress for un
+To test a `liquid` template, use the `parseFixture()` and `renderHTML()` functions in [`src/site/tests/support/index.js`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/tests/support/index.js) to create an `HTML` document. Run accessibility checks using the `axeCheck()` function in [`src/site/tests/support/axe.js`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/tests/support/axe.js).
 
-`parseFixture` takes a json fixtures path and returns a `JavaScript` object.
+Be sure to import the functions into your spec file like so:
 
-`renderHTML` takes a liquid template path and the `JavaScript` object returned by `parseFixture`, and renders an HTML document by populating the liquid template with the JSON provided. We can then run the usual mocha assertions on the result. This function uses the same code as our build process, so all of our custom liquid filters can be used.
+```js
+import { parseFixture, renderHTML } from '~/site/tests/support';
+import axeCheck from '~/site/tests/support/axe';
+
+```
+
+### Folder and File Naming Conventions
+To create a new test for a template in the `layouts` directory:
+
+1. Create a new folder in the `tests` directory under `layouts` and give the folder the same name as the template under test (but don't include the `.drupal.liquid` extension).
+2. Create a new spec file using the following naming convention: 'template name' (without the `.drupal.liquid` extension) + `.unit.spec.js`.
+3. Create a `fixtures` folder in the same directory where the `JSON` fixtures can live.
+
+Here's an illustration of this folder structure:
+
+```
+/layouts
+  /tests
+    /example_page
+      /fixtures
+      example_page.unit.spec.js
+```
+
+Use the same pattern for templates under test in other directories like `includes`, `navigation`, etc.
+
+### `parseFixture()`
+`parseFixture()` takes a `JSON` fixtures path starting from `src/` and returns a `JavaScript` object.
+
+### `renderHTML()`
+`renderHTML()` takes a `liquid` template path starting from `src/` and the `JavaScript` object returned by `parseFixture()` and renders an `HTML` document. We can then run the usual Mocha assertions on the result. This function uses the same code as our build process, so all of our custom `liquid` filters can be used.
 
-This technique can be used to generate tests of varying complexity, ranging from simple rendering sanity checks to complex logic. Since we control the JSON test data, we can easily test different scenarios.
+This technique can be used to generate tests of varying complexity, ranging from simple rendering sanity checks to complex logic. Since we control the `JSON` test data, we can easily test different scenarios.
+
+### `axeCheck()`
+`axeCheck()` takes the `HTML` document returned by `renderHTML()` and returns an array of accessibility violations.
+
+#### Disabled Axe Checks
+
+The following Axe Checks are disabled:
+
+##### `bypass`
```

Comment:
Curious about @1Copenut's take on this one

Reply:
I consulted with @1Copenut  on which rules to disable. @1Copenut would you care to elaborate on this one?

Reply:
Sure thing!

We opted to disable the `bypass` rule because it may give a false positive for lists of 4-5 links. We envision these unit tests being used for smaller content "blocks" that might trigger the rule inadvertently. 

VA.gov includes a global skip to content link that is evaluated in full page e2e tests.